src/metabase/public_settings/premium_features.clj

```diff
@@ -194,28 +214,11 @@
            :error-details (trs "Token should be a valid 64 hexadecimal character token or an airgap token.")})))
 
 (def ^{:arglists '([token])} fetch-token-status
-  "TTL-memoized version of `fetch-token-status*`. Caches API responses for 5 minutes. This is important to avoid making
-  too many API calls to the Store, which will throttle us if we make too many requests; putting in a bad token could
-  otherwise put us in a state where `valid-token->features*` made API calls over and over, never itself getting cached
-  because checks failed."
-  ;; don't blast the token status check API with requests if this gets called a bunch of times all at once -- wait for
-  ;; the first request to finish
-  (let [lock (Object.)
-        f    (memoize/ttl
-              (fn [token]
-                ;; this is a sanity check to make sure we can actually get the active user count BEFORE we try to call
```

Comment:
Note: I removed this because I moved caching deeper - it only wraps the HTTP request itself now. Therefore, a failure to get the active users count will no longer be cached.